eng/Versions.props

```diff
@@ -137,11 +137,12 @@
     <Tooling_MicrosoftCodeAnalysisAnalyzersPackageVersion>3.3.4</Tooling_MicrosoftCodeAnalysisAnalyzersPackageVersion>
     <MicrosoftCodeAnalysisExternalAccessRazorPackageVersion>$(RoslynPackageVersion)</MicrosoftCodeAnalysisExternalAccessRazorPackageVersion>
     <MicrosoftCodeAnalysisExternalAccessOmniSharpCSharpPackageVersion>$(RoslynPackageVersion)</MicrosoftCodeAnalysisExternalAccessOmniSharpCSharpPackageVersion>
-    <MicrosoftCodeAnalysisCommonPackageVersion>$(RoslynPackageVersion)</MicrosoftCodeAnalysisCommonPackageVersion>
-    <MicrosoftCodeAnalysisCSharpPackageVersion>$(RoslynPackageVersion)</MicrosoftCodeAnalysisCSharpPackageVersion>
+    <!-- Microsoft.CodeAnalysis.Common and Microsoft.CodeAnalysis.CSharp packages are consumed in source-build.
+         Package version has to match roslyn repo reference as specified in MicrosoftNetCompilersToolsetVersion -->
+    <MicrosoftCodeAnalysisCommonPackageVersion>$(MicrosoftNetCompilersToolsetVersion)</MicrosoftCodeAnalysisCommonPackageVersion>
```

Comment:
I think these are the right versions now. IMO it may be good to:
- Rename RoslynPackageVersion to something like: 
"RoslynSurfaceAreaVersion" to better reflect its usage.
- Add new dependencies to eng/Version.Details.xml for CodeAnalysis.Common and CodeAnalysis.CSharp and use their values explicitly here. Their versions would be the same as Microsoft.Net.Compilers.Toolset.